### DISCUSSION TITLE
Problem with DRM removal AZW3 file

### DISCUSSION THREAD

**Discussion by 6LifAHSwzNs52sU** [apprenticeharper/DeDRM_tools#1649] [Q&A]
Hi,

I have recently upgraded to Calibre 5.14 and DeDRM Tools 7.1.0.

I cannot seem to get the tool to work for an azw3 file (downloaded for sync with USB).

Am I doing something wrong?

I've attached the debug file.

Thanks for your help!
[tmpt73elzb8.txt](https://github.com/apprenticeharper/DeDRM_tools/files/6274590/tmpt73elzb8.txt)

**Comment by ElleKayEm** [comment]
Did you choose an E-ink Kindle (not a Fire) for the device?  Do you have that Kindle's serial number entered correctly in DeDRM's customization?

**Reply by 6LifAHSwzNs52sU** [reply]
Yes, it's an old Kindle Touch. Can you point me to the instructions on how to enter the Kindle serial number in the customization settings please? :-)

**Comment by ElleKayEm** [comment]
In calibre, Preferences > Plugins.  Then select DeDRM under File Type plugins and click Customize plugin > eInk Kindle ebooks > green plus sign.  Easiest way to get the serial number in there correctly is to copy and paste from your Content and Devices page at Amazon.  Then be sure to click OK and Apply on your way out of the dialog boxes and restart calibre before re-adding your book.

**Reply by 6LifAHSwzNs52sU** [reply]
Worked a treat. Thank you!
